prowler/lib/check/compliance.py

```diff
@@ -70,7 +71,6 @@ def update_checks_metadata_with_compliance(
                     "Recommendation": {"Text": "", "Url": ""},
                 },
                 "Categories": [],
-                "Tags": {},
```

Comment:
Why this is no longer needed?

Reply:
Because it wasn't inside the Check_Metadata_Model, I've added it

Reply:
add or remove?

Reply:
now added 😄